varsim.py

```diff
@@ -119,7 +119,9 @@ def monitor_multiprocesses(processes, logger):
             is_success = False
         else:
             logger.info("Process with pid %d finished successfully" % p.pid)
-    return is_success
+    if not is_success:
+        raise Exception('One process aborted. Please check log for details.');
```

Comment:
We need something better, probably based on multiprocessing.Pool to handle failures gracefully and also scale well with threads. Something for a different pull request.